05-foundational-skills_1.Rmd

```diff
@@ -145,4 +143,45 @@ dataedu::course_data
 
 ## Steps for working through new and unfamiliar content
 
-Need to add
\ No newline at end of file
+One skill that all great educators possess is the ability to ask questions. 
```

Comment:
super duper minor, maybe soften this just a little bit - maybe something like: "A sure sign of a great educator is the ability to ask questions."?

Reply:
great point - done!